### PR TITLE
Create jospt style

### PR DESCRIPTION
**Pull request by libora**
JSOPT style is based on AMA style - https://forums.zotero.org/discussion/22219/style-request-jospt/

### CODE REVIEW

File: dependent/jospt.csl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>JOSPT (Journal of Orthopaedic and Sports Physical Therapy)</title>
```
Comment (rmzelle) [inline dependent/jospt.csl]: Let's just go with the full title here:

`</title>Journal of Orthopaedic & Sports Physical Therapy</title>` (also notice the ampersand)

and "journal-of-orthopaedic-and-sports-physical-therapy.csl"

(see also http://www.ncbi.nlm.nih.gov/nlmcatalog/7908150)

Comment (rmzelle) [inline dependent/jospt.csl]: Oh, and to write an ampersand, use `&amp;` (ampersands need to be 'escaped' in HTML)

**Comment by adam3smith** [comment]
(before we merge -- did we check this out? The first thread in the linked post says, e.g., that JOSPT is alphabetical)

**Comment by rmzelle** [comment]
(I haven't)

**Comment by libora** [comment]
@adam3smith : now is JOSPT based on AMA and this is numerical style.

**Comment by rmzelle** [comment]
@libora, no, you haven't made any changes to this pull request since two hours ago.

**Comment by libora** [comment]
What I have to do now?
1. change full name to </title>Journal of Orthopaedic & Sports Physical Therapy</title>
2. change name of csl file to journal-of-orthopaedic-and-sports-physical-therapy.csl

**Comment by rmzelle** [comment]
Please change the title to `</title>Journal of Orthopaedic &amp; Sports Physical Therapy</title>`, and the file name, style ID, and "self" link to use "journal-of-orthopaedic-and-sports-physical-therapy" instead of "jospt".

**Comment by libora** [comment]
done, I hope without errors :-)

**Comment by rmzelle** [comment]
> (before we merge -- did we check this out? The first post in the linked thread says, e.g., that JOSPT is alphabetical)

Yeah, you're right. @libora, can you change the parent to "american-medical-association-alphabetical"? See also http://www.jospt.org/doi/pdf/10.2519/jospt.2016.6174 and http://www.jospt.org/page/authors/prepareManuscript ("References should be numbered consecutively in alphabetical order, according to author last name and initials, title, and year.").

**Comment by libora** [comment]
I hope now is everythink OK

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**